config.py

```diff
@@ -108,6 +108,15 @@ class Config(object):
     LABELS_GET_API_URL = RAS_SECURE_MESSAGE_SERVICE + 'labels?name=unread'
     THREAD_GET_API_URL = RAS_SECURE_MESSAGE_SERVICE + 'thread/{}'
 
+    RAS_FRONTSTAGE_API_HOST = os.getenv('RAS_FRONTSTAGE_API_HOST', 'localhost')
+    RAS_FRONTSTAGE_API_PORT = os.getenv('RAS_FRONTSTAGE_API_PORT', 8082)
+    RAS_FRONTSTAGE_API_PROTOCOL = os.getenv('RAS_FRONTSTAGE_API_PROTOCOL', 'http')
+    RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}/'.format(RAS_FRONTSTAGE_API_PROTOCOL,
+                                                      RAS_FRONTSTAGE_API_HOST,
+                                                      RAS_FRONTSTAGE_API_PORT)
+    GET_MESSAGES_URL = 'messages_list'
```

Comment:
MESSAGES_API_URL and MESSAGE_LIMIT now redundant

Reply:
Removed